v3.0_roadmap.adoc

```diff
@@ -0,0 +1,233 @@
+= Road map for James 3.0
+
+This document aims at dividing existing and upcoming code into the following categories :
+
+ - supported
+ - experimental
+ - unsupported
+
+== Mailbox section
+
+.Feature matrix : mailbox backend
+|===
+|Feature name |Status | Comment
+
+|JPA
+|supported
+|Already existing code
+
+|Memory
+|supported
+|Reference for testing
+
+|Cassandra
+|experimental
+|Essentially we do not know about how performance will behave
+
+|ElasticSearch
+|experimental
```

Comment:
not sure about this one, it would be could to have it supported, it looks doable, don't you think ?